torch/quantization/_quantize_script.py

```diff
@@ -22,9 +22,13 @@ def script_qconfig_dict(qconfig_dict):
 
 def _prepare_script(model, qconfig_dict, is_dynamic):
     _check_is_script_module(model)
-    if any(map(lambda x : not isinstance(x, str), qconfig_dict.keys())):
+    _check_forward_method(model)
+    if any(not isinstance(x, str) for x in qconfig_dict.keys()):
         raise ValueError('qconfig_dict should contain names(str) as keys.')
     scripted_qconfig_dict = script_qconfig_dict(qconfig_dict)
+    if not is_dynamic:
```

Comment:
Do we need this condition here? Previously we used to call this for dynamic as well.

Reply:
do we need conv bn folding for dynamic quantization?

Reply:
cc @raghuramank100